scripts/fix-unused-imports.mjs

```
@@ -0,0 +1,176 @@
+ * Examples:
+ *   node scripts/fix-unused-imports.mjs           # Compare against main/master
+ *   node scripts/fix-unused-imports.mjs main      # Compare against main
+ *   node scripts/fix-unused-imports.mjs release/1.0  # Compare against release/1.0
+ */
+
+import { execFileSync } from 'child_process';
+import { existsSync, writeFileSync, unlinkSync } from 'fs';
+import { join } from 'path';
+
+/** Valid branch name pattern (alphanumeric, hyphens, underscores, slashes, dots) */
+const VALID_BRANCH_PATTERN = /^[A-Za-z0-9._\/-]+$/;
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)